bin/cli.cjs

```diff
@@ -62,7 +62,7 @@ const showUsage = () => {
     process.exit(1);
   }
 
-  const config = require(configPath);
+  const { default: config } = await import(configPath);
```

Comment:
Actually, we can pass an array of files to `findUp` on line 56 and then consumers don't need to worry about passing an explicit `--config` flag, unless they rename their config.

```
await findUp(['playroom.config.js', 'playroom.config.mjs', 'playroom.config.cjs'], { cwd })
```

Reply:
I have not used `findUp` before - but this makes way more sense. I've also adjusted the README to reflect these changes.